pages/dkp/kaptain/2.0.0/tutorials/pipelines/kfserving-component.yaml

```diff
@@ -0,0 +1,44 @@
+name: Serve a model with KServe
```

Comment:
We don't need to include this file in the docs - I'm wondering if there is a way to ignore it during docs generation?